src/main/java/org/prebid/server/bidder/tappx/TappxBidder.java

```diff
@@ -130,6 +130,12 @@ private String buildUrl(String host, String endpoint, String tappxkey, Integer t
         }
     }
 
+    private String resolveHost(String host) {
+        return StringUtils.containsIgnoreCase(host, "http://") || StringUtils.containsIgnoreCase(host, "https://")
```

Comment:
Decided to add check exactly for `http://` or `https://`. Checking only for `.contains("http")` could lead to problems with urls like this one: `httphost.com/...`. Word `http` could be the actual part of host. But if it's a mistake of a PBJ caller - it's a PBJ caller problem :)

Reply:
We even can use `StringUtils.startsWithIgnoreCase()` to ensure the beginning of the url.
Or even: `StringUtils.startsWithAny(host.toLowerCase(), "http://", "https://")`.
Up to you, just a syntax sugar.